cmd/firectl/main.go

```diff
@@ -188,23 +200,50 @@ func main() {
 		log.Fatalf("Invalid vsock specification: %s", err)
 	}
 
+	if len(opts.FcFifoLogFile) != 0 {
+		dir, err := ioutil.TempDir(os.TempDir(), "fcfifo")
+		if err != nil {
+			log.Fatalf("Fail to create temporary directory: %v", err)
+		}
+
+		defer os.RemoveAll(dir)
+
+		if len(opts.FcLogFifo) == 0 {
```

Comment:
Is checking == 0 better than != ""?

Reply:
Naw, just style

Reply:
:+1: I don't have a strong opinion on it either way was just curious if there was a benefit I wasn't aware of.